src/index.html

```diff
@@ -6,13 +6,72 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Product Cards</title>
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="product-cards">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Product Image 1"
+          class="product-card__image"
+        />
+        <h2 class="product-card__title">Product Title 1</h2>
+        <div class="product-card__stars">
+          <span class="star">&#9733;</span>
+          <span class="star">&#9733;</span>
+          <span class="star">&#9733;</span>
+          <span class="star">&#9734;</span>
+          <span class="star">&#9734;</span>
```

Comment:
```suggestion
          <span class="stars__star">&#9733;</span>
          <span class="stars__star">&#9733;</span>
          <span class="stars__star">&#9733;</span>
          <span class="stars__star">&#9734;</span>
          <span class="stars__star">&#9734;</span>
```

Reply:
Not fixed 
<img width="928" alt="image" src="https://github.com/user-attachments/assets/d7c040f7-bf09-4749-aaf0-939096c08928">